### ISSUE TITLE
Better Labeling of Thermal Components

### ISSUE THREAD

**Issue by jrddunbr** [Electrical-Age/ElectricalAge#821]
The ```Thermal Probe``` and ```Temperature Probe``` are both probes for measuring the temperature of a ```cable``` (as quoted from the tip strip). The problem is, you can only put a ```Copper Thermal Cable``` in the ```Thermal Probe``` and you can put any cable (ex, Low Voltage, Medium Voltage, High Voltage, Very High Voltage, and the Copper Thermal Cable) in the ```Temperature Probe```.

Either we should label that the ```Thermal Probe``` is only for Copper thermal cables, or we should make it so that you can measure any cable's temperature in any temperaure/thermal probe, and call it an ```Inline Thermal Probe``` (as well as renaming the electrical probe to the ```Inline Electrical Probe``` perhaps, since they are very similar to the electrical cable measuring counterparts), since some of them are in-line with the cable, and others are just probes that connect off to the side.

Perhaps we should also clarify that power/amps can only be measured inline, while voltage and temperature can be measured to the side.

Below, find a screenshot with the "inline" measurement tools in the top, and the to the side ones at the bottom, and the voltage/amperage/power ones on the left, and the temperature/power ones on the right. The top right one will take only copper thermal cable, and the bottom right one will take any cable. The top right one should be fixed to take any cable as well.

![2017-12-16_12 55 50](https://user-images.githubusercontent.com/14130505/34073100-92836008-e260-11e7-9799-a6e129f0ee2c.png)

I'm sure there's a better term than "to the side". A different name may be advised for that as well, which describes it better.

**Comment by jrddunbr** [comment]
See Pull Request  #826, I changed the names. Looking at the code, making the ```Inline Thermal Probe``` as I refereed to it, support the electrical cables is a bit of a hassle. I guess it's not as big of a deal if it's properly labeled to show that it won't work (as I did in the PR)

**Comment by lashtear** [comment]
I'd support measuring any cable with the inline one, but I'm not sure why it currently cannot.

**Comment by jrddunbr** [comment]
Looking at the code for it, it appears that it was not intended initially - the code implies that we should not measure electrical cable temperatures, and the variables are all mixed up. I have no idea why that's considered a feature, or if it is still desired to function in that manner.

**Comment by lashtear** [comment]
Honestly I'd recommend unifying the inline/non-inline versions to one part each for electrical/thermal that can be configured to do either type of connection.  ...and again, thermal probe support of electrical cables may make certain types of designs easier, but since the current ratings are basically already known it's not particularly exploity.

**Comment by jrddunbr** [comment]
Sounds like a good idea. I'll see what I can do, still new to this.

**Comment by cm0x4D** [comment]
The two types of sensors imho makes sense:
For the inline version, the connection must be opened in order to place it and a cable has to be placed into the sensor, the other sensor just connects to the side of the cable with the drawback that serial physical values as electrical current (and electrical power, as current is needed to calculate the power) and thermal power can not be measured with that kind of probe.
I do not see a simple way to combining them into one type for each...

For the electrical cable temperature probe:
It is possible to exceed the nominal ratings of cables for a short moment. If you start doing that, you have to know the temperature of the electrical cables involved. Some folks wanted to do that and asked for support, so we added that support to the temperature probe (Measuring electrical cable temperature). Having this functionality in an "inline" probe does not make any sense, as thermal cables primary conduct heat and are not compatible with electrical cables that conduct electrical current. So aside the renaming and the new icons I would not change anything.

**Comment by jrddunbr** [comment]
I don't think that in this case it makes a difference except for being able to measure in more compact circuits. While they may be measuring different things, I think it's not incorrect to be able to measure the temperature of any cable with the inline probe. Ideally, you can measure the temperature of anything, anywhere, and for electrical stuff, measure the voltage anywhere. For current and watts, you would have a shunt, so that requires inline measurements, and for the thermal cable, the inline probe is mostly just a compact measurement tool, or to measure the watts of power conducted across it.

I think that realistically, there's no reason it shouldn't be able to measure temperature on electrical cables while being inline.

**Comment by jrddunbr** [comment]
Partially resolved in https://github.com/Electrical-Age/ElectricalAge/pull/831.
